lightwood/encoder/categorical/binary.py

```diff
@@ -2,6 +2,7 @@
 import numpy as np
 from lightwood.encoder.base import BaseEncoder
 from lightwood.helpers.constants import _UNCOMMON_WORD
+from copy import deepcopy as dc
```

Comment:
Very small nitpick, I think `dc` is a bit nondescriptive and would suggest using it as `deepcopy`

Reply:
done